<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentInformationClass</class>
 <widget class="QMainWindow" name="StudentInformationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudentInformation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 101, 120);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>42</y>
      <width>1058</width>
      <height>452</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="NameField">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="SidField">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="UserField">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="PasswordField">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RegisteredCourse">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="CompletedCourse">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>41</y>
      <width>158</width>
      <height>453</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string>SID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string>Registered Courses</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">QLabel {
    color: #3B4252;
    background-color: #F8F9FA;
    border: 1px solid #D8DEE9;
    border-radius: 8px;
    font-size: 15px;
    padding: 6px 12px;
    font-family: 'Segoe UI', 'Arial';
}</string>
       </property>
       <property name="text">
        <string>Completed Courses</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Back">
    <property name="geometry">
     <rect>
      <x>618</x>
      <y>571</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2C2C2C;       /* Dark grey background */
    color: white;                    /* Clean white text */
    border: 1px solid #444;          /* Slight edge for separation */
    border-radius: 15px;              /* Rounded corners */
    padding: 8px 16px;               /* Internal spacing */
    font-size: 14px;
    font-weight: 500;
    font-family: 'Segoe UI', sans-serif;
}

QPushButton:hover {
    background-color: #3A3A3A;       /* Slightly lighter on hover */
}

QPushButton:pressed {
    background-color: #1F1F1F;       /* Slightly darker on press */
}

QPushButton:disabled {
    background-color: #555555;
    color: #AAAAAA;
    border: 1px solid #666;
}
</string>
    </property>
    <property name="text">
     <string>Back </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1285</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
